rskj-core/src/main/java/co/rsk/peg/constants/BridgeDevNetConstants.java

```diff
@@ -169,4 +161,7 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         btcHeightWhenPegoutTxIndexActivates = 1_000_000;
         pegoutTxIndexGracePeriodInBtcBlocks = 4_320; // 30 days in BTC blocks (considering 1 block every 10 minutes)
     }
+    public static BridgeDevNetConstants getInstance() {
```

Comment:
Hey Marcos! It's necessary to make the constructor `public` to get the instance from other objects.

Reply:
Not if you are using a singleton pattern :) That's what `getInstance` method is for isn't it?

Reply:
Let's just leave as it is. We actually need to remove this devnet constants since they are not used anywhere. So I wouldn't do any extra changes. Thanks